h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -313,6 +331,113 @@ else if (aModel._output.isClassifier())
     }
   }
 
+
+
+  /**
+   * Inherit distribution and its parameters
+   * @param parms
+   */
+  private void inheritDistributionAndParms(Model.Parameters parms) {
+    if (parms instanceof GLMModel.GLMParameters) {
+      try {
+        _parms._metalearner_parameters._distribution = familyToDistribution(((GLMModel.GLMParameters) parms)._family);
+      } catch (IllegalArgumentException e) {
+        Log.warn("Stacked Ensemble is not able to inherit distribution from GLM's family " + ((GLMModel.GLMParameters) parms)._family + ".");
+      }
+    } else if (parms instanceof DRFModel.DRFParameters) {
+      inferBasicDistribution();
+    } else {
+      _parms._metalearner_parameters._distribution = parms._distribution;
+    }
+    // deal with parameterized distributions
+    switch (parms._distribution) {
+      case custom:
+          _parms._metalearner_parameters._custom_distribution_func = parms._custom_distribution_func;
+        break;
+      case huber:
+          _parms._metalearner_parameters._huber_alpha = parms._huber_alpha;
+        break;
+      case tweedie:
+        _parms._metalearner_parameters._tweedie_power = parms._tweedie_power;
+        break;
+      case quantile:
+        _parms._metalearner_parameters._quantile_alpha = parms._quantile_alpha;
+        break;
+    }
+  }
+
+  /**
+   * Inherit family and its parameters
+   * @param parms
+   */
+  private void inheritFamilyAndParms(Model.Parameters parms) {
```

Comment:
I got confused multiple time with all those params in your `inherit___` methods, especially with calls like:
```
inheritFamilyAndParms(_parms._metalearner_parameters);
...
inheritDistributionAndParms(_parms._metalearner_parameters);
```
this would mean here that `parms` and `metaParams` are the same object, so there's nothing to inherit.

Am I missing sth? Can you please clarify and possibly rename `parms` if it's intended to be only for base model params?

Reply:
My bad, this function also did set the `_dist` for the SE so that was the only reason to call it with metaParams. That was moved to different part but I kept it there so it was essentially no-op. I removed the inherit.* calls with metaParams. I also renamed `parms` to `baseModelParms` to make the intention explicit.